flexbe_core/src/flexbe_core/behavior.py

```diff
@@ -262,8 +262,8 @@ def _set_typed_attribute(self, name, value):
         if not isinstance(value, type(attr)):
             if type(attr) is int:
                 value = int(value)
-            elif type(attr) is long:
```

Comment:
<strike>
Not sure this is backwards compatible to the extend we want. 
In python2, like C++, a `long` is different from an `int`. In Py3, they're the same. </strike>

This is fine: https://www.python.org/dev/peps/pep-0237/

Reply:
We might actually remove this elif-blocks though